sys/linux/dev_rfkill.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <uapi/linux/ioctl.h>
+include <uapi/linux/rfkill.h>
+
+resource fd_rfkill[fd]
+
+syz_open_dev$rfkill(dev ptr[in, string["/dev/rfkill"]], id const[0], flags flags[open_flags]) fd_rfkill
```

Comment:
Let's keep it as `openat`. The only thing that `syz_open_dev` does today is formatting file name using that id argument. But since we don't use id in this case, the only thing it does is confusing people that there is some magic happens and producing longer reproducers.

Reply:
Hi @dvyukov,
Thanks, I'll fix it.